wait_for_postgres.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
I was wondering if this script should be included in the entrypoint script somehow ?

Reply:
It's used in docker-compose's entrypoint. Please, check referenced PR.